app.js

```diff
@@ -1,5 +1,48 @@
-// import functions and grab DOM elements
+import {countsAsAYes} from './utils.js'
+import {correctPercentage} from './utils.js'
```

Comment:
countsAsYes and correctPercentage can be imported together separated by a comma :)